app/components/manage-intermissions/index.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@glimmer/component';
+import { task } from "ember-concurrency-decorators";
+import { tracked } from "@glimmer/tracking";
+import {action} from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default class manageIntermissionsComponent extends Component {
+  @tracked intermissions = [];
+  @tracked intermissionToEdit;
+  @tracked showModal = false;
+  @service store;
+
+  constructor() {
+    super(...arguments);
+    this.fetchIntermissions.perform();
+  }
+
+  @action
+  addIntermission() {
+    this.intermissionToEdit = this.store.createRecord("intermission", {
+      startedAt: new Date()
```

Comment:
nitpicking: I think it makes sense to default to the meeting startDate if that's available